composer.json

```diff
@@ -48,11 +48,11 @@
         "symfony/security-bundle": "^4.3",
         "symfony/security-core": "^4.3",
         "symfony/security-csrf": "^4.3",
+        "symfony/string": "^5.1",
```

Comment:
Is it nesesserry whne `twig/extra-bundle` is require?

Reply:
it should be required or marked as conflict with <5.1 because otherwise you could end with string 5.0 and break because there is no cut parameter